flink-filesystems/flink-hadoop-fs/src/main/java/org/apache/flink/runtime/util/HadoopUtils.java

```diff
@@ -113,7 +113,8 @@ public static Configuration getHadoopConfiguration(org.apache.flink.configuratio
 	}
 
 	public static boolean isCredentialsConfigured(UserGroupInformation ugi, boolean useTicketCache) throws Exception {
-		if (UserGroupInformation.isSecurityEnabled()) {
+		if (UserGroupInformation.isSecurityEnabled()
+			&& ugi.getAuthenticationMethod() == UserGroupInformation.AuthenticationMethod.KERBEROS) {
```

Comment:
adding this line means the function name should not be `isCredentialsConfigured` but more like `isKerberosCredentialsConfigured`. I was wondering if we should change the public API naming.

Reply:
done, please let me know if you decide that changing that public method signature can be a problem - I'll bring back the old method name

Reply:
Hmm, I think the current method name is a bit misleading because it also returns `true` if no security has been enabled. In the current form, this method should rather be called `areKerberosCredentialsConfiguredProperlyIfKerberosAuthenticationModeIsEnabled`. This is quite a mouthful and an indicator that this method is too specific.

What about creating a `isKerberosSecurityEnabled` method and also having a `isKerberosCredentialsValid` which we only call if `isKerberosSecurityEnabled` returns true?

Reply:
+1 on this. 
also do you think we should keep the `isCredentialsConfigured` API available for 1 more release and it can simply be:
```
@Deprecated
public static boolean isCredentialsConfigured(UserGroupInformation ugi, boolean useTicketCache) throws Exception {
    return isKerberosSecurityEnabled(ugi) && isKerberosCredentialsValid(ugi, useTicketCache);
}
```